homeassistant/core.py

```diff
@@ -308,6 +300,18 @@ def async_stop(self) -> None:
 
         self.loop.stop()
 
+    @callback
+    def async_stop_handler(self, *args):
```

Comment:
Can we rename these to `async_stop()` and take 0 arguments.

Will need to update the signal handlers to properly wrap the call.